### ISSUE TITLE
UI improvements?

### ISSUE THREAD

**Issue by VasigaranAndAngel** [linsomniac/spotify_to_ytmusic#66]
I'm bringing up this topic here because the repository doesn't have discussions enabled. I'm considering enhancing the UI of this application. Do you have any suggestions for improving the UI? What are your thoughts on this? Also, I wanted to ask if you're okay with me proceeding with UI improvements?

**Comment by Yoween** [comment]
Hi! I'm the one who made the UI :)
Personally I've dropped the project, so feel free to fork the repo and upgrade the UI if you want.
It'll be up to linsomniac to decide if they use it or not but I'm pretty sure they'll accept.

**Comment by Yoween** [comment]
A good thing to do would be to rewrite the UI using PyQt but it seems to be a lot of work.
I was considering adding a dark mode, and a more clean UI in general.
I did this using the theme sv_ttk for another project : 
![image](https://github.com/linsomniac/spotify_to_ytmusic/assets/82267191/b68d8301-23eb-481a-9837-dbaa6861559a)

**Comment by VasigaranAndAngel** [comment]
> A good thing to do would be to rewrite the UI using PyQt

Yes, I share the same thoughts. I believe I can handle the PyQt coding, but UI design is not my strong suit. If someone could assist with the UI design, I could contribute to the coding. Let's see how it goes. If I find some free time, I will make some progress. Regardless of the designer's creations, I can manage the coding. (PyQt does have some limitations, but we'll address those as we push the boundaries of a cool UI.)

**some of my old project's UI parts.**

https://github.com/linsomniac/spotify_to_ytmusic/assets/72515046/f6842ed3-4395-4cb4-b42e-25398dcf46a4


https://github.com/linsomniac/spotify_to_ytmusic/assets/72515046/e2c72da6-32e5-4b7f-99d0-2d851c03f55c
